### PR TITLE
Kwxm/plc spec/bitwise conversions

### PR DESCRIPTION
**Pull request by kwxm**
[PLT-8189]  This adds the two new integer/bytestring conversions to the specification. See p35-36 of the attached PDF for a readable version.
 
[plutus-core-specification.pdf](https://github.com/IntersectMBO/plutus/files/15024318/plutus-core-specification.pdf)

### CODE REVIEW

**Comment by ramsay-t** [comment]
Are the subscripts LE and BE spelt out somewhere?

**Review by ramsay-t — APPROVED** [review]

File: doc/plutus-core-spec/cardano/builtins4.tex [hunk 1]
```diff
@@ -37,13 +43,111 @@ \subsubsection{Miscellaneous built-in functions}
...
+\begin{cases}
+  \errorX & \text{if $n<0$ or $n \geq 2^{65536}$}\\
+  \errorX & \text{if $w<0$ or $w > 8192$}\\
+  [] & \text{if $n=0$ and $0 \leq w \leq 8192$}\\
+\end{cases}
+$$
+
+\noindent  Now assume that none of the conditions above hold, so $0 < n < 2^{65536}$ and
+$0 \leq w \leq 8192$.  Since $n>0$ we can write
+$n = \sum_{i=0}^{N-1}a_{i}256^i$ with $N \geq 1$ and $a_{N-1} \ne 0$.   We then  have
```
Comment (ana-pantilie) [inline doc/plutus-core-spec/cardano/builtins4.tex:106]: Sorry, maybe it's a stupid question, but I don't understand why we can write `n` as that sum. Can you provide some more explanations?

Comment (kwxm) [reply doc/plutus-core-spec/cardano/builtins4.tex:106]: I guess it's a base-256 expansion: I'll try to explain that a bit more clearly.

**Comment by kwxm** [comment]
> Are the subscripts LE and BE spelt out somewhere?

I just had to introduce those because there wasn't room to get the full definition into the table.  I think I assumed it would be obvious what they stood for given the discussion of endianness, but it wouldn't do any harm to stick in a comment saying "(hence the subscripts LE and BE)" or something.

File: doc/plutus-core-spec/cardano/builtins4.tex (hunk 1 above)
Comment (kwxm) [reply doc/plutus-core-spec/cardano/builtins4.tex]: Oh, I forgot to say that the a_i are supposed to lie between 0 and 255. I've fixed that now.

**Pull request merged**